dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -300,6 +303,13 @@ public static ControlFlowGraph build(
         /** Does this node terminate the execution? (e.g., "System.exit()") */
         protected boolean terminatesExecution = false;
 
+        /**
+         * Provided this node terminates the execution, does the execution exit immediately? e.g.
```

Comment:
How about "Indicate whether this node terminates the execution of current method and exits the program immediately"